tidb-cloud/integrate-tidbcloud-with-netlify.md

```diff
@@ -32,32 +32,189 @@ For Dedicated Tier clusters, make sure that the traffic filter of the cluster al
 
```

Comment:
L31 的逗号似乎应该改成句号？没法加 comment。
For Dedicated Tier clusters, make sure that the traffic filter of the cluster allows all IP addresses (set to `0.0.0.0/0`) for connection. This is because Netlify deployments use dynamic IP addresses.